Service/Sources/Data/DataSource/Remote/API/ClubAttendAPI.swift

```diff
@@ -0,0 +1,97 @@
+import Moya
+import Foundation
+
+enum ClubAttendAPI {
+    case fetchAttendList(clubID: Int, date: String?, period: Period?)
+    case attendanceCreate(clubID: Int)
+    case recordeExcelPrint
+    case changeAttendStatus
+    case statusAllApply
+}
+
+extension ClubAttendAPI: GCMSAPI {
+    var domain: GCMSDomain {
+        return .attend
+    }
+
+    var urlPath: String {
+        switch self {
+        case let .fetchAttendList(clubID, date, period):
+            return "/\(clubID)"
+
+        case let .attendanceCreate(clubID):
+            return ""
+
+        case .recordeExcelPrint:
+            return "attend/excel"
+
+        case .changeAttendStatus:
+            return ""
+
+        case .statusAllApply:
+            return "batch"
+        }
+    }
+
+    var method: Moya.Method {
+        switch self {
+        case .fetchAttendList, .recordeExcelPrint:
+            return .get
+
+        case .changeAttendStatus, .statusAllApply:
+            return .patch
+
+        case .attendanceCreate:
+            return .post
+        }
+    }
+
+    var task: Task {
+        switch self {
+        case let .fetchAttendList(clubID, date, period):
```

Comment:
```suggestion
        case let .fetchAttendList(_, date, period):
```
사용 안 하는 값이라면 _ 사용하는 거 어때요?

Reply:
9cb9f396eaea9da2c80d641dd1f0ea32af935ed2
수정했습니다!